build.yaml

```diff
@@ -2292,12 +2292,16 @@ steps:
         online: true
       - name: set-test-pools-to-known-parameters
         script: /io/sql/set-test-pools-to-known-parameters.py
+        online: true
```

Comment:
This got lost in a rebase between PR merges. Shouldn't impact the existing deployments by adding online: true back to build.yaml.